docs/docs/llms/llm-intentless.mdx

```diff
@@ -0,0 +1,236 @@
+      about this feature?
+utter_faq_6:
+  - text:
+      You can contact our customer service team to have your PIN unblocked. You
+      can reach them by calling our toll-free number at 1-800-555-1234.
+```
+
+Beyond having the `utter_` prefix, the naming of the utterances is not relevant.
+
+The second step is to add
+[end-to-end stories](../training-data-format.mdx#end-to-end-training)
+to `data/e2e_stories.yml`. These stories teach the LLM about your domain, so it
+can figure out when to say what.
+
+```yaml title="data/e2e_stories.yml"
+- story: currencies
+  steps:
+    - user: How many different currencies can I hold money in?
+    - action: utter_faq_4
+
+- story: automatic transfers travel
+  steps:
+    - user: Can I add money automatically to my account while traveling?
+    - action: utter_faq_5
+
+- story: user gives a reason why they can't visit the branch
+  steps:
+    - user: I'd like to add my wife to my credit card
+    - action: utter_faq_10
+    - user: I've got a broken leg
+    - action: utter_faq_11
+```
+
+The stories and utterances in combination are used to steer the LLM. The
+difference here to the existing policies is, that you don't need to add a lot of
+intent examples to get this system going.
+
+## Testing
+
+The policy is a usual Rasa Policy and can be tested in the same way as any other
+policy.
+
+### Testing interactively
+Once trained, you can test your assistant interactively by running the following
+command:
+
+```bash
+rasa shell
+```
+
+If a flow you'd like to implement doesn't already work out of the box, you can
+add try to change the examples for the intentless policy. Don't forget that you
+can also add and edit the traditional Rasa primitives like intents, entities,
+slots, rules, etc. as you normally would. The `IntentlessPolicy` will kick in
+only when the traditional primitives have low confidence.
+
+### End-to-End stories
+
+As part of the beta, we're also releasing a beta version of a new End-To-End
+testing framework. The `rasa test e2e` command allows you to test your bot
+end-to-end, i.e. from the user's perspective. You can use it to test your bot in
+a variety of ways, including testing the `IntentlessPolicy`.
+
+To use the new testing framework, you need to define a set of test cases in a
+test folder, e.g. `tests/e2e_test_stories.yml`. The test cases are defined in a
+similar format as stories are, but contain the user's messages and the bot's
+responses. Here's an example:
+
+```yaml title="tests/e2e_test_stories.yml"
+test_cases:
+  - test_case: transfer charge
+    steps:
+      - user: how can I send money without getting charged?
+      - utter: utter_faq_0
+      - user: not zelle. a normal transfer
+      - utter: utter_faq_7
+```
+
+**Please ensure all your test stories have unique names!** After setting the
+beta feature flag for E2E testing in your current shell with
+`export RASA_PRO_BETA_E2E=true`, you can run the tests with
+`rasa test e2e -f tests/e2e_test_stories.yml`
```

Comment:
We should remove this section since there is a different part of the docs dedicated to end-to-end testing.